app/src/main/java/de/tum/in/tumcampusapp/component/ui/news/KinoDetailsFragment.java

```diff
@@ -137,6 +142,15 @@ private void createKinoHeader(LinearLayout rootView) {
         year.setOnClickListener(view -> Toast.makeText(context, R.string.year, Toast.LENGTH_SHORT).show());
         trailer.setOnClickListener(view -> showTrailer());
 
+        // Setup "Buy/Show ticket" button according to ticket status for current event
+        if (isBooked){
+            ticket.setText("Show ticket");
+            ticket.setOnClickListener(view -> showTicket());
+        } else{
```

Comment:
You should run autoformat before commiting any changes: https://stackoverflow.com/a/16580200/409460

Reply:
I see. I will do that next time!